### PR TITLE
brightnessctl: add Malayalam translation

### PR DESCRIPTION
**Pull request by sanbabyfrancis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ml/linux/brightnessctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> മോണിറ്ററിന്റെ പ്രകാശം നിയന്ത്രിക്കാനുള്ള യൂട്ടിലിറ്റി.
```
Comment (kbdharun) [inline pages.ml/linux/brightnessctl.md:3]: Can you update this line alone to better align with the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/brightnessctl.md).

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution, I have a suggestion for this page.

**Comment by kbdharun** [comment]
I have a request can you update the words for Malayalam (ml) in [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) in a new PR. As it will be helpful for reviewers and other translators.

File: pages.ml/linux/brightnessctl.md (hunk 1 above)
Comment (sanbabyfrancis) [reply pages.ml/linux/brightnessctl.md]: Updated

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**